### DISCUSSION TITLE
Is it possible to use ImageMagick to change the thumbnail in the exifdata of an image?

### DISCUSSION THREAD

**Discussion by Kjelldor** [ImageMagick/ImageMagick#6189] [Help]
### ImageMagick version

7

### Operating system, version and so on

Win11 v22H2

### Description

I've been trying this in Magick.NET and have been researching whether or not I could add this functionality, however, I would love to know if it is even available in ImageMagick itself (if not, I might just make a seperate tool that just changes thumbnails).

If it is available, I would love to know where to look in the code to see an example.

Kind regads,

Kjelldor.

**Comment by fmw42** [comment]
As far as I know, Imagemagick does not edit EXIF data. Use EXIFTOOL.

**Comment by snibgo** [comment]
I think fmw42 means Imagemagick does *not* edit EXIF data.

**Reply by fmw42** [reply]
Yes. I it was a typo. I meant "not" edit EXIF data. My spell checker keeps changing my words!
